src/auth/src/credentials.rs

```diff
@@ -13,8 +13,8 @@
 // limitations under the License.
 
 mod api_key_credentials;
-// Export API Key factory function and options
-pub use api_key_credentials::{ApiKeyOptions, create_api_key_credentials};
+// Export API Key credentials builder and options
+pub use api_key_credentials::{ApiKeyOptions, Builder as ApiKeyCredentialsBuilder};
```

Comment:
I do not think we want to reexport `ApiKeyCredentialsBuilder`.  We don't do that for any of the other builders.  And I think we can remove `ApiKeyOptions`, nothing uses it:

```suggestion
pub use api_key_credentials;
```